resources/profiles/FlyingBear/machine/fdm_klipper_common.json

```diff
@@ -16,11 +16,11 @@
   "cooling_tube_length": "5",
   "cooling_tube_retraction": "91.5",
   "default_filament_profile": [
-      "FlyingBear Generic PLA"

```

Comment:
This change may has issues as there is no `FlyingBear PLA` filament.
Better to keep "FlyingBear Generic PLA" here

Reply:
Hello. I have already modified the relevant files and all the other filament files will be modified to "FlyingBear XX" and I will be adding other filaments as well, so do I need to submit them all at once?

Reply:
Yes, please consolidate all changes into one PR. 
Also, let's not rename or delete existing filaments/processes/etc., as that would invalidate existing user configs based on them. You can add new filaments instead.

Reply:
" let's not rename or delete existing filaments/processes/etc., as that would invalidate existing user configs based on them. You can add new filaments instead."If I modify the code inside InfiMech.json so that it and the modified file names are aligned with InfiMech.json, will this ensure that it won't cause conflicts. If I want to delete some files completely

Reply:
> " let's not rename or delete existing filaments/processes/etc., as that would invalidate existing user configs based on them. You can add new filaments instead."If I modify the code inside FlyingBear.json so that it and the modified file names are aligned with FlyingBear.json, will this ensure that it won't cause conflicts. If I want to delete some files completely

Unfortunately, the proposed changes will still lead to compatibility issues, and I'd like to clarify that I'm not referring to mere conflicts. My primary concern is that these changes will prevent existing **USER** profiles from being loaded in Orca. For instance, if a user has created a profile named 'AAA filament' based on the 'FlyingBear Generic PLA' profile, this user will no longer be able to use the 'AAA filament' profile if the 'FlyingBear Generic PLA' profile is deleted.These changes are disruptive for Orca users with FlyingBear machines. Therefore, we should avoid making such changes